awx/main/management/commands/register_queue.py

```diff
@@ -104,7 +104,7 @@ def handle(self, **options):
             changed3 = False
             (ig, created, changed1) = self.get_create_update_instance_group(queuename, inst_per, inst_min)
             if created:
-                print(six.text_type("Creating instance group {}".format(ig.name)))
+                print(six.text_type("Creating instance group {}").format(ig.name))
```

Comment:
In python3, I think we probably don't actually need `six.text_type` at all anymore :)  The more I look at this pull request, the more I wonder if this all just works now.

Reply:
I'm going to go back through at some point now that py3 merged and removed all of our extraneous `six.text_type` usage.